index.html

```diff
@@ -12,22 +12,28 @@
         href="https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;600&family=Titillium+Web:wght@200;400;600&display=swap">
     <link rel="stylesheet" href="/styles/reset.css" />
     <link rel="stylesheet" href="/styles/global.css" />
-    <title>Web App</title>
+    
 </head>
 
 <body>
     <header>
-        <img src="/assets/html_css_js.png">
-        <h1>Web App</h1>
+        <section id="search-site">
```

Comment:
In this spot, I think you could have just used the header tag as the section with the id of search-site.  Remember that header, main, footer, and section are all just divs that we give names to for semantic reasons.